apps/api/src/domain/models/audio-visual/audio-item/repositories/arango-audio-item-query-repository.ts

```diff
@@ -249,6 +254,46 @@ export class ArangoAudioItemQueryRepository implements IAudioItemQueryRepository
         await this.database.query({ query, bindVars });
     }
 
+    // why are the speaker initials here?
+    async translateLineItem(
+        id: AggregateId,
+        { languageCode, text, inPointMilliseconds, outPointMilliseconds }: TranslationLineItemDto
+    ) {
+        const newMultilingualTextItem = new MultilingualTextItem({
+            languageCode,
+            text,
+            // TODO add this to the event payload
+            role: MultilingualTextItemRole.freeTranslation,
+        });
+
+        const query = `
+        FOR doc IN @@collectionName
+        FILTER doc._key == @id
+
+        let updatedItems = (
+            FOR item IN doc.transcript.items
+            let updatedItem = item.inPointMilliseconds == @inPointMilliseconds && item.outPointMilliseconds == @outPointMilliseconds ? MERGE(item,{ text: { items: APPEND(item.text.items,@newItem) }}) : item
+            return updatedItem
+        )
+
+        update doc with {
```

Comment:
We might want to break out a generic `translateMultilingualTextProperty` to the base query builder. But it could be tricky with nested properties. For now, let's see how breaking out a general `translateName` or `translateTextProperty` works at the top level.